openpype/tools/publisher/widgets/widgets.py

```diff
@@ -198,12 +198,20 @@ def __init__(self, parent=None):
         self.setLayoutDirection(QtCore.Qt.RightToLeft)
 
 
+class SaveBtn(PublishIconBtn):
+    """Save context and instances information."""
+    def __init__(self, parent=None):
+        icon_path = get_icon_path("save")
+        super(SaveBtn, self).__init__(icon_path, parent)
+        self.setToolTip("Save changes (Ctrl + S)")
+
+
 class ResetBtn(PublishIconBtn):
     """Publish reset button."""
     def __init__(self, parent=None):
         icon_path = get_icon_path("refresh")
         super(ResetBtn, self).__init__(icon_path, parent)
-        self.setToolTip("Refresh publishing")
+        self.setToolTip("Refresh publishing (Ctrl + R)")
```

Comment:
The shortcuts likely need their labels set from the OS respective key sequence string for it so that on mac it's still accurate?

Reply:
Changed `Ctrl` to `Command` for MacOS